config/config.go

```diff
@@ -48,6 +48,11 @@ var (
 		"proxy-authenticate":                {},
 		"proxy-authorization":               {},
 		"www-authenticate":                  {},
+
+		// Added by SigV4. X-Amz-Content-Sha256 is not included since it may be
+		// explicitly provided to override request signing.
```

Comment:
What would be the usecase?

Reply:
I don't have one off the top of my head. It's referenced from the [Sign docs](https://pkg.go.dev/github.com/aws/aws-sdk-go@v1.34.29/aws/signer/v4#Signer.Sign). Happy to add it in, maybe it was intended more for developers and not users.

Reply:
This seems to be for developers and it would make prometheus not be able to make valid signs for the requests.

Reply:
Yeah, this is clearly for developers and not for users. I'm going to add it to the list here.